sonar-batch/src/main/java/org/sonar/batch/issue/IssueFilters.java

```diff
@@ -59,4 +73,145 @@ public boolean accept(DefaultIssue issue) {
       return false;
     }
   }
+
+  private Issue toIssueForIssueFilter(final String componentKey, final BatchReport.Issue rawIssue) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "toIssueForIssueFilter" is 42 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This method has 140 lines, which is greater than the 100 lines authorized. Split it into smaller methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS138)